integration-tests/testconfig/vrfv2plus/overrides/ilja_pavlovs.toml

```diff
@@ -0,0 +1,6 @@
+[Network]
```

Comment:
I think we can delete this file

Reply:
@iljapavlovs ok, I can remove it. Next time you want to test something on Github CI you can commit such file and run use `test_config_override_path` to run a workflow with it.